pkg/volume/gce_pd/attacher.go

```diff
@@ -173,6 +173,8 @@ func (attacher *gcePersistentDiskAttacher) WaitForAttach(spec *volume.Spec, devi
 				// A device path has successfully been created for the PD
 				glog.Infof("Successfully found attached GCE PD %q.", pdName)
 				return path, nil
+			} else {
+				glog.V(4).Infof("could not verifying GCE PD (%q) is attached, device path is empty", pdName)
```

Comment:
How frequent is this going to be?

Also "verifying" -> "verify"

Reply:
it is every second. The timeout is 10 minutes. Actually I think 10 minutes might be a  bit of long because it will block any other operations on this device if attach fails.

Reply:
Do you think every second may be too frequent and could cause log spam?